lib/utils/report.js

```diff
@@ -109,3 +109,26 @@ module.exports = function report(problem) {
 
 	result.warn(warningMessage, warningProperties);
 };
+
+/**
+ * @param {import('postcss').Node | undefined} node
+ * @param {number | undefined} index
+ * @param {number | undefined} endIndex
+ * @returns {number | undefined}
+ */
+function getStartLine(node, index, endIndex) {
+	if (node === undefined) return undefined;
+
+	// HACK: The `rangeBy` method has been added since PostCSS 8.4.0,
+	//       while the `node` may be produced by PostCSS 8.3.x or lower
+	//       that doesn't have the method.
+	//       See https://github.com/stylelint/stylelint/issues/5766
+	if (typeof node.rangeBy !== 'function') {
+		console.warn(
+			'It appears different PostCSS versions are used in your dependency tree. Please consider updating PostCSS to use one version.',
+		);
+		node.rangeBy = postcss.Node.prototype.rangeBy.bind(node);
```

Comment:
It may work without `bind` now, but I've added it just in case. Because it's hard to predict how `rangeBy` is and will be used inside PostCSS.

Or, is there some disadvantage to adding `bind`?

Reply:
The prototype itself does not bind, so we don't need to do that neither.

Reply:
I see. Remove via 02a9c7518. 👍🏼